server/server/models/user.py

```diff
@@ -69,3 +70,9 @@ def is_active(self, request):
                                 .filter_by(status="Active").one().id
                 return True
         return False
+
+    def get_role(self, request):
+        """Return string with user role"""
+        role_obj = request.dbsession.query(Role).get(self.role_id)
```

Comment:
You need to use SQLAlchemy's capabilities to handle relations. There is `roles = relationship("Role", foreign_keys=(role_id,))` (but I have no idea why it's in plural form) and you should use it.